src/account/azext_account/generated/_help.py

```diff
@@ -0,0 +1,79 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['account subscription'] = """
+    type: group
+    short-summary: account subscription
+"""
+
+helps['account subscription create-subscription'] = """
+    type: command
+    short-summary: The operation to create a new WebDirect or EA Azure subscription.
+    examples:
+      - name: createSubscription
+        text: |-
+               az account subscription create-subscription --billing-account-name \\
+               "0aa27f2b-ec7f-5a65-71f6-a5ff0897bd55:ae0dae1e-de9a-41f6-8257-76b055d98372_2019-05-31" \\
+               --billing-profile-name "27VR-HDWX-BG7-TGB" --cost-center "135366376" --display-name \\
+               "Contoso MCA subscription" --owner object-id=973034ff-acb7-409c-b731-e789672c7b32 \\
+               --sku-id "0001" --invoice-section-name "JGF7-NSBG-PJA-TGB"
+"""
+
+helps['account subscription create-subscription-in-enrollment-account'] = """
+    type: command
+    short-summary: Creates an Azure subscription
```

Comment:
what's the differnece from above in short-description. seems the extension is purely generated